src/applications/vaos/new-appointment/components/TypeOfFacilityPage.jsx

```diff
@@ -23,50 +25,39 @@ const initialSchema = {
     facilityType: {
       type: 'string',
       enum: Object.keys(FACILITY_TYPES).map(key => FACILITY_TYPES[key]),
-    },
-  },
-};
-
-const uiSchema = {
-  facilityType: {
-    'ui:title':
-      'You’re eligible to see either a VA provider or community care provider for this type of care.',
-    'ui:widget': 'radio',
-    'ui:options': {
-      labels: {
-        [FACILITY_TYPES.VAMC]: (
-          <>
-            <span className="vads-u-display--block vads-u-font-size--lg vads-u-font-weight--bold">
-              VA medical center or clinic
-            </span>
-            <span className="vads-u-display--block vads-u-font-size--sm">
-              Go to a VA medical center or clinic for this appointment
-            </span>
-          </>
-        ),
-        [FACILITY_TYPES.COMMUNITY_CARE]: (
-          <>
-            <span className="vads-u-display--block vads-u-font-size--lg vads-u-font-weight--bold">
-              Community care facility
-            </span>
-            <span className="vads-u-display--block vads-u-font-size--sm">
-              Go to a community care facility near your home
-            </span>
-          </>
-        ),
-      },
+      enumNames: ['VA medical center or clinic', 'Community care facility'],
     },
   },
 };
 
 const pageKey = 'typeOfFacility';
-const pageTitle = 'Choose where you want to receive your care';
 
 export default function TypeOfFacilityPage({ changeCrumb }) {
   const featureBreadcrumbUrlUpdate = useSelector(state =>
     selectFeatureBreadcrumbUrlUpdate(state),
   );
 
+  const pageTitle = useSelector(state => getPageTitle(state, pageKey));
+
+  const uiSchema = {
+    facilityType: {
+      'ui:title': pageTitle,
+      'ui:widget': 'radio', // Required
+      'ui:webComponentField': VaRadioField,
+      'ui:options': {
+        classNames: 'vads-u-margin-top--neg2',
+        showFieldLabel: false,
+        labelHeaderLevel: '1',
+        descriptions: {
+          [FACILITY_TYPES.VAMC]:
+            'Go to a VA medical center or clinic for this appointment',
+          [FACILITY_TYPES.COMMUNITY_CARE]:
+            'Go to a community care facility near your home',
+        },
```

Comment:
I'm wondering if having the label placed within the ui:options just above the descriptions can provide greater context.
```suggestion
      'ui:options': {
        classNames: 'vads-u-margin-top--neg2',
        showFieldLabel: true,
        labelHeaderLevel: '1',
        labels: {
          [FACILITY_TYPES.VAMC]: 'VA medical center or clinic',
          [FACILITY_TYPES.COMMUNITY_CARE]: 'Community care facility',
        },
        descriptions: {
          [FACILITY_TYPES.VAMC]:
            'Newer text from figma file',
          [FACILITY_TYPES.COMMUNITY_CARE]:
            'Newer text from figma file',
        },
```

Reply:
Ah that looks much better.